client/components/popover/menu.jsx

```diff
@@ -20,6 +21,7 @@ var PopoverMenu = React.createClass( {
 
 	getDefaultProps: function() {
 		return {
+			autoPosition: true,
```

Comment:
I opted to go for consistency with what `<Popover />` uses as its default for `autoPosition`. Do you think it makes sense for both of them to be the same, or not necessarily?